.ddev/config.yaml

```
@@ -7,7 +7,7 @@ router_https_port: "443"
 xdebug_enabled: false
 additional_hostnames: []
 additional_fqdns: []
-mariadb_version: "10.3"
+mariadb_version: "10.5"
```

Will apply this in a dedicated commit to be able to pick the other changes to all branches.
```suggestion
mariadb_version: "10.3"
```

---

See https://github.com/GsTYPO3/ddev-typo3-src/commit/bbdc0401a0a0fbcc8fae74ba093a2f1606033f0c